java/src/jmri/jmrix/cmri/serial/sim/configurexml/ConnectionConfigXml.java

```diff
@@ -33,6 +33,11 @@ protected void getInstance() {
     }
 
     @Override
+    protected void getInstance(Object object) {
```

Comment:
I thought I replied to this, but I don't see it here...

I think you may be right.  The interesting part of this is that the jmri.jmrix.AbstractConnectionConfigXml class, from which the jmri.jmrix.configurexmlAbstract{Serial,Simulator,Network}ConnectionConfigXml classes are derived, has a version of getInstance that takes a ConnectionConfig object as a parameter.

What I don't know is if the version in getInstance was created before or after the ones in the derived classes.